src/dependabot_alerts/core.py

```diff
@@ -1,184 +1,66 @@
-              }
-              severity
-              vulnerableVersionRange
-            }
-          }
-        }
-      }
-    }
-  }
-}
-"""
-
-    vulns = []
-    cursor = None
-    has_next_page = True
-
-    while has_next_page:
-        result = gh.query(
-            query=query, variables={"organization": organization, "cursor": cursor}
+    @classmethod
+    def make(cls, alert_dict):
+        return cls(
+            repo=safe_get(alert_dict, ["repository", "full_name"]),
+            ecosystem=safe_get(alert_dict, ["dependency", "package", "ecosystem"]),
+            package=safe_get(alert_dict, ["dependency", "package", "name"]),
         )
-        page_info = result["organization"]["repositories"]["pageInfo"]
-        cursor = page_info["endCursor"]
-        has_next_page = page_info["hasNextPage"]
-
-        for repo in result["organization"]["repositories"]["nodes"]:
-            alerts = repo["vulnerabilityAlerts"]["nodes"]
-
-            if alerts:
-                repo_name = repo["name"]
-                vulnerable_packages = set()
-
-                for alert in alerts:
-                    sa = alert["securityAdvisory"]
-                    sv = alert["securityVulnerability"]
-                    number = alert["number"]
-                    package_name = sv["package"]["name"]
-
-                    if package_name in vulnerable_packages:
-                        continue
-                    vulnerable_packages.add(package_name)
-
-                    pr = None
-
-                    dep_update = alert["dependabotUpdate"]
-                    if dep_update and dep_update["pullRequest"]:
-                        pr = dep_update["pullRequest"]["url"]
 
-                    vuln = Vulnerability(
-                        repo=repo_name,
-                        created_at=alert["createdAt"],
-                        ecosystem=sv["package"]["ecosystem"],
-                        number=number,
-                        package_name=sv["package"]["name"],
-                        pr=pr,
-                        severity=sv["severity"],
-                        title=sa["summary"],
-                        url=f"https://github.com/{organization}/{repo_name}/security/dependabot/{number}",
-                        version_range=sv["vulnerableVersionRange"],
-                    )
-                    vulns.append(vuln)
 
-    return vulns
+class GitHub:
+    def __init__(self, run):
+        self._run = run
+
+    def alerts(self, organization):
+        try:
+            result = self._run(
+                [
+                    "gh",
+                    "api",
+                    "--paginate",
+                    f"/orgs/{organization}/dependabot/alerts?state=open",
+                ],
+                check=True,
+                capture_output=True,
+                text=True,
+            )
+        except CalledProcessError as err:  # pragma: no cover
+            print(err.stdout)
+            print(err.stderr)
```

Comment:
I've changed this to print any stdout from GitHub CLI to stdout and any stderr from GitHub CLI to stderr, and to prefix each line with `GitHub CLI stdout> ` or `GitHub CLI stderr> `:

https://github.com/hypothesis/dependabot-alerts/blob/8046836b6eda14dcd8091037f80708c2079c9689/src/dependabot_alerts/core.py#L54-L59

Example (trying to get alerts for a user rather than an organization fails):

```terminal
$  tox -qqe dev --run-command 'dependabot-alerts seanh'
GitHub CLI stdout> {"message":"Not Found","documentation_url":"https://docs.github.com/rest/dependabot/alerts#list-dependabot-alerts-for-an-organization"}
GitHub CLI stderr> gh: Not Found (HTTP 404)
Traceback (most recent call last):
  File "/home/seanh/GitHub/hypothesis/dependabot-alerts/.tox/dev/bin/dependabot-alerts", line 8, in <module>
    sys.exit(cli())
             ^^^^^
  File "/home/seanh/GitHub/hypothesis/dependabot-alerts/.tox/dev/lib/python3.12/site-packages/dependabot_alerts/cli.py", line 22, in cli
    repos = github.alerts(organization)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/seanh/GitHub/hypothesis/dependabot-alerts/.tox/dev/lib/python3.12/site-packages/dependabot_alerts/core.py", line 43, in alerts
    result = self._run(
             ^^^^^^^^^^
  File "/home/seanh/.pyenv/versions/3.12.0/lib/python3.12/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['gh', 'api', '--paginate', '/orgs/seanh/dependabot/alerts?state=open']' returned non-zero exit status 1.
```